test/unit/lookups/esri_test.rb

```diff
@@ -26,6 +28,23 @@ def test_query_for_geocode_with_token_for_storage
       res
   end
 
+  def test_query_for_geocode_with_client_credentials_for_storage
+    token_data = {
+      access_token: "xxxxx",
+      expires_in: 7200
+    }
+    stub_request(:post, "https://www.arcgis.com/sharing/rest/oauth2/token").to_return(:body => token_data.to_json)
```

Comment:
@TrangPham what mechanism can I use to stub that method? It looks like `Test::Unit` doesn't have a `.stub` method like RSpec or MiniTest do.